### PR TITLE
kubel-exec-vterm-pod sets correct default-directory with tramp

### PR DESCRIPTION
**Pull request by kingsuntw**

### CODE REVIEW

File: kubel.el [hunk 1]
```diff
@@ -929,24 +929,21 @@ P can be a single number or a localhost:container port pair."
...
+  (let* ((dir-prefix (or
+                      (when (tramp-tramp-file-p default-directory)
+                        (with-parsed-tramp-file-name default-directory nil
+                          (format "%s%s:%s@%s|" (or hop "") method user host))) ""))
+         (con-pod (kubel--get-container-under-cursor))
          (container (car con-pod))
          (pod (cdr con-pod))
-         (command (format "%s exec %s -c %s -i -t -- /usr/bin/env bash" (kubel--get-command-prefix) pod container)))
-    (with-current-buffer (vterm (concat "*kubel:vterm:" container "@" pod "*"))
-      (set-process-sentinel vterm--process #'kubel-in-vterm-kill)
```
Comment (abrochard) [inline kubel.el:947]: why don't we need the sentinel anymore?

**Comment by abrochard** [comment]
Thank you for submitting this! I'm not super familiar with vterm. Would you be able to explain your changes a bit more please?

File: kubel.el (hunk 1 above)
Comment (kingsuntw) [reply kubel.el:947]: it's already handled by the following code:
https://github.com/akermu/emacs-libvterm/blob/b44723552f86407d528c4a6c8057382c061b008e/vterm.el#L736

Comment (abrochard) [reply kubel.el:947]: Wonderful! Thank you for letting me know!

**Comment by kingsuntw** [comment]
When function `kubel-exec-vterm-pod` was called, the variable `default-directory` was not set to TRAMP remote path. So when function `find-file` was called, the local path was prompted.

I would like the remote path been prompted, just like function `kubel-exec-eshell-pod` did.

**Review by abrochard — APPROVED** [review]

**Pull request merged**